project/src/main/webapp/voting_edu.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
```

Comment:
nit: It might not be necessary to mention "voting" in the HTML file name. Perhaps change the name to `education.html`, which is more similar to the other files. However this is totally your call! Feel free to ignore this.